### ISSUE TITLE
[FEATURE] Add sticky navigation bar

### ISSUE THREAD

**Issue by anamika7153** [MilanCommunity/Milan#839]
### Description

I'm in GSSoC'23.

When We move downwards, navbar also gets scrolled up and when we scroll up then also navbar doesn't display. We have to move all the way up to access the Navbar. Navbar should either be sticky or displayed when scolled up.

### Screenshots

# Laptop Screen

<img width="960" alt="2023-06-10" src="https://github.com/tamalCodes/Milan/assets/66116440/7bd2eaab-b6df-4714-96f8-9d13503bd3ca">

# Mobile Screen

![WhatsApp Image 2023-06-10 at 16 20 57](https://github.com/tamalCodes/Milan/assets/66116440/73428c8b-10d5-428d-b22c-836f912249a7)

### 🥦 Browser

Google Chrome

### Checklist ✅

- [X] I checked and didn't find similar issue
- [X] I have read the [Contributing Guidelines](https://github.com/tamalCodes/Milan/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no)

**Comment by tamalCodes** [comment]
Hi there, currently we are waiting on #774 for implementing the same. If they do not reply or do not want to work on it, will assign this to you. For now I would love to keep this on hold.

**Comment by anamika7153** [comment]
> Hi there, currently we are waiting on #774 for implementing the same. If they do not reply or do not want to work on it, will assign this to you. For now I would love to keep this on hold.

sure :)
